server/tso_test.go

```diff
@@ -38,14 +39,20 @@ func (s *testTsoSuite) TearDownSuite(c *C) {
 }
 
 func sendRequest(c *C, conn net.Conn, msgID uint64, request *pdpb.Request) {
```

Comment:
I think we can use these help functions in other places to reduce the codes too.
Outer no need to care msgpb.Message.